kiosk/src/Components/AddingGame.tsx

```diff
@@ -55,7 +55,11 @@ const AddingGame: React.FC<IProps> = ({ kiosk }) => {
             try {
                 const newKioskCode: string = await generateKioskCodeAsync();
                 setKioskCode(newKioskCode);
-                await addGameToKiosk(newKioskCode);
+                try {
+                    await addGameToKiosk(newKioskCode);
+                } catch (error) {
+                    console.log("Unable to add game to kioksk");
```

Comment:
```suggestion
                    console.log("Unable to add game to kiosk");
```

Do we not need a user-facing error here? If this exception is hit, will it just not work with no explanation?

Reply:
Yes, we should have a user-facing error here. Correct, it will just not work with no explanation.

 I needed to change this from throwing an error, however, because if the step of polling for a game code was throwing an error and thus getting caught in the outer catch and setting `renderQrCode` to false, it causes problems more often on the adding game page than needed. 

I'm not sure how to approach the error handling in the best way, though, that is the most impactful for the user. I am wanting to sync with the designers about error handling. Also, I'm going to tackle the error handling when I complete this issue: https://github.com/microsoft/pxt-arcade/issues/5295. I am going to prioritize this next week.